plugins/GSdx/GSTextureCache.cpp

```diff
@@ -379,11 +379,6 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 			int ww = (int)(fr.left + m_renderer->GetDisplayRect().width());
```

Comment:
That else branch is necessary to fix issues at VP2 on custom resolutions. Yes,  it's a hack of (it lowers the size of scaling to make it sufficient with the buffet size)